### PR TITLE
Add MonkeyC & Jungle Lexers

### PR DESCRIPTION
**Pull request by pzl**
This adds lexers for a programming language (monkeyc) and a mini resource-definition language (jungle).

Both are inventions of the [Garmin ecosystem](https://developer.garmin.com/connect-iq/monkey-c/) called connect IQ, to program their wearables and gadgets.

### CODE REVIEW

**Comment by alecthomas** [comment]
I think the tests need to be regenerated.

File: lexers/j/jungle.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package j
+
+import (
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+var MonkeyC = internal.Register(MustNewLexer(
```
Comment (alecthomas) [inline lexers/j/jungle.go]: This should be called Jungle shouldn't it?

Comment (pzl) [reply lexers/j/jungle.go]: good catch, copy/paste typo. Will fix these and clean up the history

**Comment by alecthomas** [comment]
Brilliant, thanks!

**Pull request merged**